pyproject.toml

```
@@ -5,7 +5,7 @@ build-backend = "flit_core.buildapi"
 [project]
 # Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
 name = "ansys-mapdl-core"
-version = "0.65.dev0"
+version = "0.64.1"
```

```suggestion
version = "0.65.dev0"
```
Was it modified on purpose?

---

It wasn't :). Good catch!